modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx

```diff
@@ -135,6 +135,7 @@ export function ModalAddDefaultSortColumn({
 							emptyStateMessage={Liferay.Language.get(
 								'there-are-no-columns-added-in-this-view-yet'
 							)}
+							id="modal-add-default-sort-column"
```

Comment:
Hi @Artartur. Please, rename the id to `objectViewModalAddDefaultSortColumn`.

Reply:
Done!